### PR TITLE
Introduce Settings.includeFlat(Iterable)

### PR DESCRIPTION
**Pull request by donat**
Follow-up PR for https://github.com/gradle/gradle/pull/19135

### CODE REVIEW

**Comment by donat** [comment]
@bot-gradle test rFM

**Comment by donat** [comment]
@bot-gradle test rFM

**Comment by donat** [comment]
@gradle/execution can I get a 👍  here and on #19135?

File: subprojects/architecture-test/src/changes/accepted-public-api-changes.json [hunk 1]
```diff
@@ -23,6 +23,22 @@
...
+            "changes": [
+                "Method added to interface"
+            ]
+        },
+        {
+            "type": "org.gradle.api.initialization.Settings",
+            "member": "Method org.gradle.api.initialization.Settings.includeFlat(java.lang.String[])",
+            "acceptation": "Adding new Settings#includeFlat(Iterable)",
+            "changes": [
+                "METHOD_ABSTRACT_NOW_DEFAULT"
```
Comment (lptr) [inline subprojects/architecture-test/src/changes/accepted-public-api-changes.json:40]: Is this a good message here? I thought we used plain English.

**Review by lptr — APPROVED** [review]
LGTM.

File: subprojects/architecture-test/src/changes/accepted-public-api-changes.json (hunk 1 above)
Comment (donat) [reply subprojects/architecture-test/src/changes/accepted-public-api-changes.json:40]: I got it from https://github.com/gradle/gradle/commit/e74330dac12b29b923872bc42670cfb64d50eaa4

**Pull request merged**